modules/tensor_mechanics/include/postprocessors/AveragePointSeparation.h

```diff
@@ -0,0 +1,55 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralPostprocessor.h"
+
+// Forward Declarations
+class AveragePointSeparation;
+
+template <>
+InputParameters validParams<AveragePointSeparation>();
+
```

Comment:
```suggestion
```

remove this!

Reply:
This file is no longer in the commit now.